### ISSUE TITLE
Label truncation is leading to unexpected behaviour when used inside the tableview cell.

### ISSUE THREAD

**Issue by kidsid49** [hsusmita/ResponsiveLabel#29]
As shown in screenshot below setAttributedTruncationToken works fine but when I tap on the cell "drawText" gets called and looks like it resizes the label again. here it is looks like. 
I don't have any idea why this is happening?
Any clue?

Other than that my text includes a newline but its seems that spacing is gone when I tap on the cell.
Check the third screenshot.

<img width="384" alt="truncation" src="https://cloud.githubusercontent.com/assets/3293006/12171475/3d622344-b570-11e5-99f0-382c0ed67b5d.png">
<img width="385" alt="expended" src="https://cloud.githubusercontent.com/assets/3293006/12171476/3dc85b28-b570-11e5-8224-f6305e9a1fa0.png">

<img width="384" alt="screen shot 2016-01-07 at 6 59 02 pm" src="https://cloud.githubusercontent.com/assets/3293006/12171557/d254012a-b570-11e5-84cc-1c6abe3addb3.png">

**Comment by hsusmita** [comment]
@kidsid49 This seems very interesting. Are you sure the problem is only for text with new line? I will try to dig in.

**Comment by hsusmita** [comment]
@kidsid49 Are you changing text or attributes on cell tap? I have implemented same thing in my demo project. It works fine there. Can you give me more input?

**Comment by kidsid49** [comment]
Nope. I just find that on cell tap it calls drawText thats why this is happening.

**Comment by kidsid49** [comment]
@hsusmita  here is my code in cellRowAtIndexPath -       

 NSString *expansionToken = NSLocalizedString(@"MORE_TRUNCATION", nil);
        NSMutableAttributedString *attribString = [[NSMutableAttributedString alloc]initWithString:expansionToken attributes:@{NSForegroundColorAttributeName:[[GSUtil colorFromHexString:@"#0C101E"] colorWithAlphaComponent:0.5],RLHighlightedForegroundColorAttributeName:[[GSUtil colorFromHexString:@"#0C101E"] colorWithAlphaComponent:0.3], NSFontAttributeName:storyCell.showReviewLabel.font,RLTapResponderAttributeName:expensionTap}];
        [storyCell.showReviewLabel setAttributedTruncationToken:attribString];
        [storyCell.showReviewLabel setText:reviewText withTruncation:YES];
